src/QuadForm/Lattices.jl

```diff
@@ -966,6 +966,46 @@ function restrict_scalars(L::AbsLat)
   return ZLat(Vabs, Mabs)
 end
 
+@doc Markdown.doc"""
+    restrict_scalars_with_map(L) -> 
+       Tuple{ZLat, Hecke.SpaceRes}}
+
+Given a hermitian lattice `L`, return the $\mathbb Z$-lattice obtained by 
+restriction of scalars, together with the map `f` for extending scalars back.
+"""
+function restrict_scalars_with_map(L)
```

Comment:
```suggestion
function restrict_scalars_with_map(L::AbsLat)
```
We need the type signatures for your functions and documentations.